services/user.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    new_user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        new_user.email = email
+
+    if first_name:
+        new_user.first_name = first_name
+
+    if last_name:
+        new_user.last_name = last_name

```

Comment:
You can move it to new function and remove duplicate of code inside update_user and create_user